frontend/src/scenes/actions/ActionsTable.js

```diff
@@ -108,91 +110,66 @@ export function ActionsTable() {
         },
     ]
 
-    const Caption = () => {
-        return (
-            <>
-                {featureFlags['actions-ux-201012'] && (
-                    <div>
-                        Actions can retroactively group one or more raw events to help provide consistent analytics.{' '}
-                        <a href="https://posthog.com/docs/features/actions" target="_blank">
-                            <QuestionCircleOutlined />
-                        </a>
-                    </div>
-                )}
-                {!featureFlags['actions-ux-201012'] && (
-                    <div>
-                        Actions are PostHog’s way of easily sorting through events. Actions consist of one or more
-                        events that you have decided to put into a manually-labelled bucket. They're used in Insights
-                        and Cohorts.
-                        <br />
-                        <br />
-                        <a href="https://posthog.com/docs/features/actions" target="_blank" rel="noopener noreferrer">
-                            See documentation
-                        </a>
-                    </div>
-                )}
-            </>
-        )
-    }
-
     return (
         <div>
-            <PageHeader title="Actions" caption={<Caption />} />
-            {featureFlags['actions-ux-201012'] && (
+            <div>
                 <div>
-                    <div className="tutorial-container">
-                        <div className="t-element">
-                            <div>
-                                <img src={imgGrouping} alt="" />
-                            </div>
-                            <div>
-                                <div className="title">Multiple grouping</div>
-                                <div className="description">Group multiple sets of events into a single action.</div>
-                            </div>
+                    Actions can retroactively group one or more raw events to help provide consistent analytics.{' '}
```

Comment:
"consistent" is not the best word here - implies that not using them is inconsistent.